include/picongpu/plugins/transitionRadiation/executeParticleFilter.hpp

```diff
@@ -47,6 +47,6 @@ namespace picongpu::plugins::transitionRadiation
             = pmacc::traits::HasIdentifier<typename T_Species::FrameType, transitionRadiationMask>::type::value;
 
         if constexpr(hasTransitionRadiationFilter)
-            particles::Manipulate<picongpu::plugins::transitionRadiation::GammaFilter, T_Species>{}(currentStep);
+            particles::manipulate<picongpu::plugins::transitionRadiation::GammaFilter, T_Species>(currentStep);
```

Comment:
When was this renamed to a lower case `manipulate`?

Reply:
solves include issues by using signature from `Manipulate.def` and the change is equivalent. The new form is btw. better to read